src/tests/ftest/cart/iv/cart_iv_one_node.py

```diff
@@ -109,6 +109,11 @@ def setUp(self):
         self.utils = CartUtils()
         self.env = self.utils.get_env(self)
 
+    def tearDown(self):
+        """ Tear down """
+        super(TestWithoutServers, self).tearDown()
```

Comment:
(pylint-bad-super-call) Bad first argument 'TestWithoutServers' given to super()